### ISSUE TITLE
IE11 Checkout Error; Indefinite loading; Cant complete check out

### ISSUE THREAD

**Issue by AndyJAllen** [magento/magento2#23837]
### Preconditions (*)

1. Magento 2.3.1
2. Paypal Payflow Pro
3. Internet Explorer

### Steps to reproduce (*)

1. Use Internet Explorer
2. Check out with credit card entry using Payflow Pro

### Expected result (*)

1. Successful checkout
2. Confirmation page with order number

### Actual result (*)

1.Never-ending loader:
![image](https://user-images.githubusercontent.com/19735797/61716546-6750e880-ad2d-11e9-84ca-084a1f18ba1d.png)

2. IE Console:
![image](https://user-images.githubusercontent.com/19735797/61716430-1ccf6c00-ad2d-11e9-8ff1-4a974cd356fb.png)

Error:
SCRIPT65535: Argument not optional
list-mixin.min.js (3,1)

**Comment by AndyJAllen** [comment]
Pertinent thread on stackexchange:
https://magento.stackexchange.com/questions/274405/magento-paypal-recaptcha-module-ie11-errors

**Comment by ktpl-rammohan** [comment]
@magento give me 2.3-develop instance

**Comment by magento-engcom-team** [comment]
Hi @rammohan-krish. Thank you for your request. I'm working on Magento 2.3-develop instance for you

**Comment by magento-engcom-team** [comment]
Hi @rammohan-krish, here is your [Magento instance](https://i-23837-2-3-develop.instances.magento-community.engineering).
Admin access: [https://i-23837-2-3-develop.instances.magento-community.engineering/admin](https://i-23837-2-3-develop.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`
Instance will be terminated in up to 3 hours.

**Comment by AndyJAllen** [comment]
FYI the solution suggested on the Stack Exchange thread that I linked above worked for me...so consider patching the Paypal Captcha module.

```
/**
 * Overrides default window.clearTimeout() to catch errors from iframe and reload Captcha.
 */
clearTimeout: function (timeoutID) {
    var captcha = captchaList.getCaptchaByFormId(this.formId);

    if (captcha !== null) {
        captcha.refresh();
    }
    clearTimeout(timeoutID);
}
```

**Comment by ktpl-rammohan** [comment]
@AndyJAllen Thanks for the information, will check and confirm the issue.

**Comment by ktpl-rammohan** [comment]
@magento give me 2.2-develop instance

**Comment by magento-engcom-team** [comment]
Hi @rammohan-krish. Thank you for your request. I'm working on Magento 2.2-develop instance for you

**Comment by ktpl-rammohan** [comment]
@magento give me 2.2-develop instance

**Comment by magento-engcom-team** [comment]
Hi @rammohan-krish. Thank you for your request. I'm working on Magento 2.2-develop instance for you

**Comment by magento-engcom-team** [comment]
:white_check_mark: Confirmed by @rammohan-krish 
 Thank you for verifying the issue. Based on the provided information internal tickets `MC-18783` were created

**Issue Available**: @rammohan-krish, _You will be automatically unassigned. Contributors/Maintainers can claim this issue to continue. To reclaim and continue work, reassign the ticket to yourself._

**Comment by AndyJAllen** [comment]
I believe there are multiple issues with the Payment module that need resolved to account for Internet Explorer as well.... For example, in Magento_Payment\templates\transparent\iframe.phtml  its using window.location by itself; but its known that in certain versions of IE it needs to be within an anonymous function to work properly. I've been getting a surprising amount of customers that keep experiencing payment issues with Internet Explorer.

**Comment by magento-engcom-team** [comment]
Hi @AndyJAllen. 

Thank you for your report and collaboration!

The issue was fixed by Magento team. The fix was delivered into `magento/magento2:2.3-develop` branch(es).
Related commit(s):
 - [6d642f83ad5551d5adc52930a227e66b87ddf480](https://github.com/magento/magento2/commit/6d642f83ad5551d5adc52930a227e66b87ddf480)

The fix will be available with the upcoming `2.3.4` release.

**Comment by AndyJAllen** [comment]
404 on your commit link.
